Neuro/Interactions/InteractionsHandler.cs

```diff
@@ -71,8 +71,15 @@ public void UseTarget(IUsable usable)
             // case OpenDoorConsole simpleDoor:
             //     break;
 
-            // case PlatformConsole platform:
-            //     break;
+            case PlatformConsole platform:
+                if (platform.Platform.IsLeft && MovementHandler.Instance.ForcedMoveDirection.x == 1.0f) {
```

Comment:
@Alexejhero is a simple check like this ok or would you prefer checking the angle (see ladder)

Reply:
it should be ok, NN can only respond with 8 directions, you could even change the ladder to a simple check like this.